dev-vcs/guitar/guitar-0.2.1~git.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="A QT git client"
+DESCRIPTION="Git GUI Client
+
+* Open Source, Free Software
+* Run on Windows, macOS and Linux
+* Written in C++
+* Powered by Qt 5"
+HOMEPAGE="https://soramimi.github.io/Guitar/"
+COPYRIGHT="2017 S.Fuchita"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="828a6f934d86f8eb42a81c12fdb6ae52063a6835"
+SOURCE_URI="https://github.com/soramimi/Guitar/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="dd6d5f54856dbb6911993344bad2e684564654a04ddf01ee1e016d7018654178"
+SOURCE_DIR="Guitar-$srcGitRev"
+SOURCE_FILENAME="Guitar-$portVersion-$srcGitRev.tar.gz"
+ADDITIONAL_FILES="guitar.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64 ?arm ?ppc"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	guitar$secondaryArchSuffix = $portVersion
+	app:Guitar
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:file
+	cmd:git
+	lib:libcrypto$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcrypto$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:git
```

Comment:
```diff
-	cmd:git
```
I think that `cmd:git` is only needed in `REQUIRES`.
I'll kick a build without it in `BUILD_PREREQUIRES` to check this.

Reply:
I can confitm that you can remove `cmd:git` from `BUILD_PREREQUIRES`. (I even recommend doing so.)
The resulting package is identical to the one we get if we have `cmd:git` in `BUILD_PREREQUIRES`.
(Of course there is one difference, but it is just in the metadata in `.PackageInfo` file.)